src/components/bookComponent.css

```diff
@@ -0,0 +1,132 @@
+.book * {
+  background-color: white;
+}
+
+.book {
+  background-color: white;
+  height: 7.625rem;
+  border: 1px solid #e8e8e8;
+  display: flex;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+}
```

Comment:
- [ ] In each book `<div>` element, should have the tracking graph on the right side of the text '65% Completed'. Kindly let's add it as given in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏